app/controllers/policy_classes_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PolicyClassesController < PlanningApplicationsController
+  before_action :set_planning_application
+  before_action :set_policy_class, only: %i[show update destroy]
+
+  def part; end
+
+  def new
+    @part = params[:part]
+
+    redirect_to part_new_policy_assessment_path(@planning_application), alert: "Please choose one of the policy parts" if @part.blank?
+  end
```

Comment:
was trying to test this error message but after clicking "Continue" without selecting a part I got this error -:
![image](https://user-images.githubusercontent.com/9452321/134502075-5e48f601-61dd-4a37-8320-12bd45ab9eaa.png)

![image](https://user-images.githubusercontent.com/9452321/134502053-4dbb0d14-62e3-4147-a45d-980212620bbc.png)

Reply:
I gather the error should be in create instead?

Reply:
I forgot to update the route, thanks!